src/mesh/interpolation/hermite_spline.cxx

```diff
@@ -28,9 +28,15 @@
 #include <vector>
 
 HermiteSpline::HermiteSpline(int y_offset, Mesh *mesh)
-    : Interpolation(y_offset, mesh), h00_x(localmesh), h01_x(localmesh), h10_x(localmesh),
-      h11_x(localmesh), h00_z(localmesh), h01_z(localmesh), h10_z(localmesh),
-      h11_z(localmesh) {
+    : Interpolation(y_offset, mesh),
+      h00_x(localmesh, CELL_CENTRE, DIRECTION::X, DIRECTION::YOrthogonal, DIRECTION::Z),
+      h01_x(localmesh, CELL_CENTRE, DIRECTION::X, DIRECTION::YOrthogonal, DIRECTION::Z),
+      h10_x(localmesh, CELL_CENTRE, DIRECTION::X, DIRECTION::YOrthogonal, DIRECTION::Z),
+      h11_x(localmesh, CELL_CENTRE, DIRECTION::X, DIRECTION::YOrthogonal, DIRECTION::Z),
+      h00_z(localmesh, CELL_CENTRE, DIRECTION::X, DIRECTION::YOrthogonal, DIRECTION::Z),
+      h01_z(localmesh, CELL_CENTRE, DIRECTION::X, DIRECTION::YOrthogonal, DIRECTION::Z),
+      h10_z(localmesh, CELL_CENTRE, DIRECTION::X, DIRECTION::YOrthogonal, DIRECTION::Z),
+      h11_z(localmesh, CELL_CENTRE, DIRECTION::X, DIRECTION::YOrthogonal, DIRECTION::Z) {
```

Comment:
Similarly, can these just use the defaults? I'm sure there's a reason you've done this, but it's not obvious to me, sorry!